README.md

```diff
@@ -191,6 +314,8 @@ batch_metric_data = [one_metric_data, one_metric_data]
 # Send list of data immediately
 wavefront_sender.send_metric_now(batch_metric_data)
 ```
+***Note***: If your metric name has a bad character, that character is replaced with a `-`.
```

Comment:
Maybe: ...the proxy replaces that character with a `-`. 
?

Reply:
Checked with Aliaksandrand he mentioned that this is done by the SDK itself, not the proxy.